### ISSUE TITLE
Can't parse webhooks in OpenAPI 3.1 spec structured via Redocly

### ISSUE THREAD

**Issue by samanthawritescode** [mkon/openapi_contracts#89]
I use Redocly to bundle and split my OpenAPI 3.1.0 spec, which includes webhooks documented under the [`webhook` field](https://spec.openapis.org/oas/v3.1.0#oasWebhooks).

Based on the first line in the README, and no mention of limited support for OpenAPI 3.1.0, I expected this to work.
> Currently supports OpenAPI documentation in the structure as used by [Redocly](https://github.com/Redocly/create-openapi-repo), but should also work for single file schemas.

Instead, when I try to parse my spec ([here's the main file](https://github.com/persona-id/persona-openapi/blob/main/openapi.yaml)), I receive this error: 

```
RuntimeError:
  Unknown file: #<Pathname:webhooks/account.created.yaml>
# /usr/local/bundle/gems/openapi_contracts-0.13.0/lib/openapi_contracts/parser/transformers/pointer.rb:29:in `transform_pointer'
# /usr/local/bundle/gems/openapi_contracts-0.13.0/lib/openapi_contracts/parser/transformers/pointer.rb:20:in `transform_refs'
# /usr/local/bundle/gems/openapi_contracts-0.13.0/lib/openapi_contracts/parser/transformers/pointer.rb:5:in `call'
# /usr/local/bundle/gems/openapi_contracts-0.13.0/lib/openapi_contracts/parser.rb:59:in `block in transform_objects!'
# /usr/local/bundle/gems/openapi_contracts-0.13.0/lib/openapi_contracts/parser.rb:59:in `each'
# /usr/local/bundle/gems/openapi_contracts-0.13.0/lib/openapi_contracts/parser.rb:59:in `transform_objects!'
# /usr/local/bundle/gems/openapi_contracts-0.13.0/lib/openapi_contracts/parser.rb:58:in `block in transform_objects!'
# /usr/local/bundle/gems/openapi_contracts-0.13.0/lib/openapi_contracts/parser.rb:58:in `each_value'
# /usr/local/bundle/gems/openapi_contracts-0.13.0/lib/openapi_contracts/parser.rb:58:in `transform_objects!'
# /usr/local/bundle/gems/openapi_contracts-0.13.0/lib/openapi_contracts/parser.rb:58:in `block in transform_objects!'
# /usr/local/bundle/gems/openapi_contracts-0.13.0/lib/openapi_contracts/parser.rb:58:in `each_value'
# /usr/local/bundle/gems/openapi_contracts-0.13.0/lib/openapi_contracts/parser.rb:58:in `transform_objects!'
# /usr/local/bundle/gems/openapi_contracts-0.13.0/lib/openapi_contracts/parser.rb:51:in `block in file_to_data'
# <internal:kernel>:90:in `tap'
# /usr/local/bundle/gems/openapi_contracts-0.13.0/lib/openapi_contracts/parser.rb:50:in `file_to_data'
# /usr/local/bundle/gems/openapi_contracts-0.13.0/lib/openapi_contracts/parser.rb:24:in `block in parse'
# /usr/local/bundle/gems/openapi_contracts-0.13.0/lib/openapi_contracts/parser.rb:21:in `each'
# /usr/local/bundle/gems/openapi_contracts-0.13.0/lib/openapi_contracts/parser.rb:21:in `each_with_object'
# /usr/local/bundle/gems/openapi_contracts-0.13.0/lib/openapi_contracts/parser.rb:21:in `parse'
# /usr/local/bundle/gems/openapi_contracts-0.13.0/lib/openapi_contracts/parser.rb:8:in `call'
# /usr/local/bundle/gems/openapi_contracts-0.13.0/lib/openapi_contracts/doc.rb:14:in `parse'
```

**Comment by samanthawritescode** [comment]
Hey @mkon thanks for the extremely prompt response! I tried from that branch and it does look like that fixed it.

**Comment by mkon** [comment]
Parsing is solved with #90, however testing webhook payloads is not in scope yet.
